examples/other/background_subtraction.py

```diff
@@ -0,0 +1,189 @@
+        self.uniform_buffer.update_range()
+
+class BackGroundRemovedImage(gfx.Image):
+    pass
+
+@register_wgpu_render_function(BackGroundRemovedImage, BackGroundRemovedImageMaterial)
+class BackGroundRemovedImageShader(ImageShader):
+    def __init__(self, wobject):
+        super().__init__(wobject)
+        material = wobject.material
+
+    def get_code(self):
+        code = super().get_code()
+        code = code.replace(
+            """
+    let value = sample_im(varyings.texcoord.xy, sizef);
+""",
+            """
+    let new_size = sizef.xy / pow(2., f32(u_material.background_level));
+    let texcoords_u = vec2<f32>(varyings.texcoord.xy * new_size);
+    let texcoords_img = vec2<f32>(varyings.texcoord.xy * sizef.xy);
+    var f: vec2<f32> = fract(min(texcoords_u + 0.5, vec2<f32>(new_size)));
+    var f_img: vec2<f32> = fract(min(texcoords_img + 0.5, vec2<f32>(sizef.xy)));
+
+    // Get the background value via bilinear interpolation
+    var b0: vec4<f32> = mix(
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_u + vec2<f32>(-0.5, -0.5)), 
+            i32(u_material.background_level)),
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_u + vec2<f32> (0.5, -0.5)), 
+            i32(u_material.background_level)),
+        f.x,
+    );
+
+    var b1: vec4<f32> = mix(
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_u + vec2<f32>(-0.5, 0.5)), 
+            i32(u_material.background_level)),
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_u + vec2<f32>( 0.5, 0.5)), 
+            i32(u_material.background_level)),
+        f.x,
+    );
+    let background = mix(b0, b1, f.y);
+
+    // Get the image value via bilinear interpolation
+    var v0: vec4<f32> = mix(
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_img + vec2<f32>(-0.5, -0.5)), 
+            0),
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_img + vec2<f32> (0.5, -0.5)), 
+            0),
+        f_img.x,
+    );
+
+    var v1: vec4<f32> = mix(
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_img + vec2<f32>(-0.5, 0.5)), 
+            0),
+        textureLoad(t_img, 
+            vec2<i32>(texcoords_img + vec2<f32>( 0.5, 0.5)), 
+            0),
+        f_img.x,
+    );
+    var value: vec4<f32> = mix(v0, v1, f_img.y);
+    if f32(u_material.background_level) != 0.0 {
+        value = vec4<f32>(
+            (value.rgb - background.rgb),
+            value.a
+        );
+    }
+"""
+        )
+        return code
+
+
+
+
+image = BackGroundRemovedImage(
+    gfx.Geometry(grid=image_texture), 
+    BackGroundRemovedImageMaterial(
+        clim=(0, 255), 
+        interpolation="nearest"
+        )
+)
+scene.add(image)
+
+current_background_index = 0
+current_image_index = 0
+
+def draw_imgui():
+    global current_background_index
+    global current_image_index
+    global im, image_texture
+
+    imgui.new_frame()
+
+    imgui.set_next_window_size((300, 0), imgui.Cond_.always)
```

Comment:
The label is cut off (for me), this fixes it. 
```suggestion
    imgui.set_next_window_size((400, 0), imgui.Cond_.always)
```